### PR TITLE
Added sample chart in Readme

### PR DESCRIPTION
**Pull request by Pratikshadange2005**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,10 @@ The data is refreshed daily to provide up-to-date insights into repository activ
 This project’s lineage is illustrated below:
 
 ![Lineage Diagram](transform/docs/dbt_dag.png)
+
+### Cumulative Evaluation of Stars for Top 5 Repos Over Time
+
+![Cumulative Stars for Top 5 Repositories Over Time](transform/docs/Monthly_star_count.png)
+	
+This chart shows the cumulative star count over time for the top 5 repositories with the highest stars each year.
```
Comment (edsioufi) [inline README.md]: Minor nit: This shouldn't be top 5 repos as we're using a sample containing only these 5 repos (I think).

File: Notebook/gharchive.ipynb
Comment (edsioufi) [inline Notebook/gharchive.ipynb:1]: The python notebook shows an error in github, did you save it after your last run to the latest version?
You can check the error by choosing "View File" under the menu options at the top right of this file in the "Files Changed" section

Comment (edsioufi) [reply Notebook/gharchive.ipynb:1]: Another thing is not to use in the notebook an absolute path to your database (contains your username etc.), but replace it with a relative path like `../github_stars.db` for example

**Review by edsioufi — CHANGES_REQUESTED** [review]
Great progress, with a few things to fix before merging!

File: README.md (hunk 1 above)
Comment (Pratikshadange2005) [reply README.md]: done

**Pull request merged**